src/main/java/seedu/address/logic/commands/CommandType.java

```diff
@@ -0,0 +1,37 @@
+package seedu.address.logic.commands;
+/**
+ * Enum representing the type of commands.
+ */
+public enum CommandType {
```

Comment:
hmm, might be better to separate the words with underscore? similar to constants.
(convention for constants)
e.g ADD_TASK

Reply:
Alright, will do

Reply:
Good Job bro